packages/sources/svelte/preprocess.js

```diff
@@ -0,0 +1,62 @@
+const svelte = require('svelte/compiler');
```

Comment:
This script seems to me a bit too "manual", could we consider moving to a more "automated" way ? 
I would recommand using rollup to manage all of this. This is pretty convenient for libraries. 
Also, there is a [component-template](https://github.com/sveltejs/component-template) proprosed by Rich Harris and maintained by Svelte core team. 

If you don't mind, I'd like switching closer to this template before merging this PR. I'll have a look on it. 

Any insights on this ?

Reply:
Yes feel free Terence! Moreover if we can follow practices from Rich Harris before merging, for sure it would be better!

Reply:
Is component-template ready for a package `@vtmn/svelte` that will contain several components? Like we did with existing packages for web components & React components.

Reply:
It's not ready if we take it "as it is", but it can surely be adapted in the context !

Reply:
Done in https://github.com/Decathlon/vitamin-web/pull/97/commits/2bb5c733553771d8708ade8ad4afc5be42312c53 ✅